src/components/CohortBuilder/Categories/Category.js

```diff
@@ -19,15 +54,78 @@ const Title = styled('h3')`
   color: #404c9a;
 `;
 
-const Category = ({ title, color }) => (
-  <Container>
-    <Title> {title}</Title>
-  </Container>
+const CategoryButton = styled(Column)`
+  align-items: center;
+`;
+
+const Category = ({
+  title,
+  children,
+  color,
+  toggleDropdown,
+  isDropdownVisible,
+  setDropdownVisibility,
+  toggleExpanded,
+  toggleExpandedDropdown,
+  setActiveIndex,
+  activeIndex,
+  setExpanded,
+  showExpanded,
+  setAppliedFilters,
+  appliedFilters,
+}) => (
+  <React.Fragment>
+    <Dropdown
+      {...{
+        multiLevel: true,
+        onOuterClick: () => {
+          setExpanded(false);
+          setDropdownVisibility(false);
+        },
+        isOpen: isDropdownVisible,
+        onToggle: toggleDropdown,
+        setActiveIndex,
+        activeIndex,
+        setExpanded,
+        showExpanded,
+        showArrow: false,
+        items: items.map((item, i) => (
+          <CategoryRow active={appliedFilters.includes(i)} title={item.name} />
+        )),
+        expandedItems: items.map((item, i) => (
+          <Filter
+            onCancel={toggleExpandedDropdown}
+            onBack={toggleExpanded}
+            onApply={active => {
+              toggleExpanded();
+              const newAppliedFilters = active
+                ? union([i], appliedFilters)
+                : appliedFilters.filter(af => af != i);
+              setAppliedFilters(newAppliedFilters);
+            }}
+          >
+            {item.expanded}
+          </Filter>
+        )),
+        ContainerComponent: Container,
+        OptionsContainerComponent: Options,
+        ItemWrapperComponent: ItemWrapper,
+      }}
+    >
+      <CategoryButton>
+        {children}
+        <Title> {title}</Title>
+      </CategoryButton>
+    </Dropdown>
+  </React.Fragment>
 );
 
 Category.propTypes = {
   title: PropTypes.string.isRequired,
   // /color: PropTypes.string.isRequired,
 };
 
-export default Category;
+export default compose(
+  withDropdownMultiPane,
+  compose(withState('appliedFilters', 'setAppliedFilters', [])),
```

Comment:
I have a feeling the inference of the applied state will end up being a little more complex than this, so I'd leave this out for now and hardcode `active` in `CategoryRow` as `true`